keyless/circuit-tests/Cargo.toml

```diff
@@ -14,8 +14,13 @@ rust-version = { workspace = true }
 
 [dependencies]
 anyhow = { workspace = true }
+aptos-crypto.workspace = true
 aptos-keyless-common = { workspace = true }
+ark-bn254 = "0.4.0"
+ark-ff = "0.4.2"
 base64 = { workspace = true }
 itertools.workspace = true
+rand.workspace = true
+rand_chacha = "0.3.1"
```

Comment:
Are you saying there's no way to import `rand_chacha` only for this specific crate? Seems a little weird to add it to the workspace `Cargo.toml`, since it's only used for tests.

Reply:
it looks like moving `rand_chach` to `[dev-dependencies]` works.